extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/internal/DmxBridgeHandler.java

```diff
@@ -0,0 +1,237 @@
+            case CHANNEL_MUTE:
+                if (command instanceof OnOffType) {
+                    isMuted = ((OnOffType) command).equals(OnOffType.ON);
+                } else {
+                    logger.warn("command {} not supported in channel {}:mute", command.getClass(), this.thing.getUID());
+                }
+                break;
+            default:
+                logger.warn("Channel {} not supported in bridge {}", channelUID.getId(), this.thing.getUID());
+        }
+    }
+
+    /**
+     * get a DMX channel from the bridge
+     *
+     * @param channel
+     *            a BaseChannel that identifies the requested channel
+     * @param thing
+     *            the Thing that requests the channel to track channel usage
+     * @return
+     *         a Channel object
+     */
+    public Channel getDmxChannel(BaseChannel channel, Thing thing) {
+        return universe.registerChannel(channel, thing);
+    }
+
+    /**
+     * remove a thing from all channels in the universe
+     *
+     * @param thing
+     *            the thing that shall be removed
+     */
+    public void unregisterDmxChannels(Thing thing) {
+        universe.unregisterChannels(thing);
+    }
+
+    /**
+     * get the universe associated with this bridge
+     *
+     * @return
+     *         the DMX universe id
+     */
+    public int getUniverseId() {
+        return universe.getUniverseId();
+    }
+
+    /**
+     * rename the universe associated with this bridge
+     *
+     * @param universeId
+     *            the new DMX universe id
+     */
+    protected void renameUniverse(int universeId) {
+        universe.rename(universeId);
+    }
+
+    @Override
+    public void thingUpdated(Thing thing) {
+        updateConfiguration();
+    }
+
+    /**
+     * open the connection to send DMX data to
+     */
+    protected abstract void openConnection();
+
+    /**
+     * close the connection to send DMX data to
+     */
+    protected abstract void closeConnection();
+
+    /**
+     * send the buffer of the current universe
+     */
+    protected abstract void sendDmxData();
+
+    /**
+     * install the sending and updating scheduler
+     */
+    protected void installScheduler() {
+        if (senderJob != null) {
+            uninstallScheduler();
+        }
+        if (refreshTime > 0) {
+            senderJob = scheduler.scheduleAtFixedRate(packetSender, 1, refreshTime, TimeUnit.MILLISECONDS);
+            logger.trace("started scheduler for thing {}", this.thing.getUID());
+        } else {
+            logger.info("refresh disabled for thing {}", this.thing.getUID());
+        }
+    }
+
+    /**
+     * uninstall the sending and updating scheduler
+     */
+    protected void uninstallScheduler() {
+        if (senderJob != null) {
+            if (!senderJob.isCancelled()) {
+                senderJob.cancel(true);
+            }
+            senderJob = null;
+            closeConnection();
```

Comment:
done. I changed the abstract closeConnection to a full method, checking if ThingStatusDetail is NONE, which is only used here. If it is NONE, the thing status is not updated.